runtime/compilers/rillv1/parse_rillyaml.go

```diff
@@ -40,6 +42,7 @@ type VariableDef struct {
 
 // rillYAML is the raw YAML structure of rill.yaml
 type rillYAML struct {
```

Comment:
I think you will also need to add handling of `mock_users:` here, which I believe the frontend currently adds to enable testing security policies locally. (Eventually it would probably be good to expose those over the APIs so the UI can avoid parsing them separately, but that's for another PR.)